README.md

```diff
@@ -42,13 +43,7 @@ Build deployable web page. This will generate the production web site.
 awestruct -P production
 ```
 
-Upload a web page to github.io. If everything went ok, web page will be updated automatically.
-```
-rsync -Pr --protocol=28 --delete-after _site/* jbosstm@filemgmt.jboss.org:www_htdocs/jbosstm/
-```
-
-
-If you did any changes on the development branch. Go back and push them to GitHub.
+To upload the (new) production web site, run the following command. If everything went OK, the new web site is updated automatically.
 ```
-git push origin develop
```

Comment:
What was your rationale for removing these two lines?

Reply:
I am not sure on the "correctness" [1] of the current sequence but essentially I guess:
```If you did any changes on the development branch. Go back and push them to GitHub.

`git push origin develop`
```
covers the case where some local change was made (that was not sent as a pull request and merged).

I don't actually think I would tend to build it locally anyway and I think I would tend to download and unzip a _site.zip artifact from CI to update the site.

[1] if it stays, regarding my correctness comment I suppose the push to the `develop` branch should be before making the site live - plus I suppose it's more correct (even for the website) to pull request any changes needed rather than push it direct.